src/configuration.cpp

```diff
@@ -149,7 +172,7 @@ chimera::CompiledConfiguration::CompiledConfiguration(
 , ci_(ci)
 {
     // Start out by setting the binding name to the default.
```

Comment:
Nit: This comment needs to be updated to something like "Start out by setting the binding name to an empty string.".